tests/cpgrid/adapt_cpgrid_test.cpp

```diff
@@ -207,6 +208,11 @@ void markAndAdapt_check(Dune::CpGrid& coarse_grid,
         const auto& grid_view = coarse_grid.leafGridView();
         Dune::MultipleCodimMultipleGeomTypeMapper<Dune::CpGrid::LeafGridView> adaptMapper(grid_view, Dune::mcmgElementLayout());
 
+        auto itMin = std::min_element((data.back() -> global_cell_).begin(),  (data.back()-> global_cell_).end());
+        auto itMax = std::max_element((data.back() -> global_cell_).begin(),  (data.back() -> global_cell_).end());
+        BOOST_CHECK_EQUAL( *itMin, 0);
+        BOOST_CHECK_EQUAL( *itMax, data.front()-> size(0) -1);
```

Comment:
Maybe change this test to work also for grid with inactive cells?
You could check that each value is between zero and the maximum cartesian index possible.

Reply:
There are checks in opm-grid/tests/cpgrid/inactiveCells_lgr_test.cpp, e.g. [here](https://github.com/OPM/opm-grid/blob/36069f06e5fe321c2dd161a7614dc95be5d05978/tests/cpgrid/inactiveCell_lgr_test.cpp#L182). However, OPM/opm-grid#761 intends to remove all the implementation details, in particular the previous check. 

In this case, we can replace data.front()-> size(0)-1 by grid.logicalCartesianSize()[0]*...[1]*....*[2]